packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5574,24 +4951,27 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Specify joining via audio (option: pstn), video, screenshare
+   * Creates a media connection to the server. Media connection is required for sending or receiving any audio/video.
+   * In order to send audio/video, call publishTracks() after addMedia() resolves.
+   *
    * @param {Object} options A configurable options object for joining a meeting
```

Comment:
you could update this to be `AddMediaOptions`

Reply:
done